mdgen/docker/integration_tests/features/feature/launch.feature

```diff
@@ -0,0 +1,27 @@
+Feature: The java app
+
+  Background:
+    Given the java app exists
+    And the login credentials exist
+
+  Scenario: Launching the java app
+    When I run the java executable with no parameters
+    Then I see that the application complains about missing parameters
+
+  Scenario Outline: Launching the original use case java app
```

Comment:
Maybe we should rename these scenarios so it's clear what the original use case and the new use case is?